src/actions/getPOI.action.js

```diff
@@ -0,0 +1,46 @@
+import * as actionTypes from './actionTypes'
+import Api from './../utils/apiWrapper'
+
+function pois(pois) {
+  return { type: actionTypes.LOAD_POIS, payload: pois }
+}
+
+function poiById(poi) {
+  return { type: actionTypes.LOAD_POI_ID, payload: poi }
+}
+
+function poiByMapYear(poi) {
+  return { type: actionTypes.LOAD_POI_MAP_YEAR, payload: poi }
+}
+
+function poiByStoryId(poi) {
+  return { type: actionTypes.LOAD_POI_STORY_ID, payload: poi }
+}
+
+export function getPois() {
+  return dispatch => {
+    return Api.getPOIs().then(poi => dispatch(pois(poi)))
+  }
+}
+
+export function getPoiById(id) {
+  return dispatch => {
+    return Api.getPOI(Number(id)).then(pois => dispatch(poiById(pois)))

```

Comment:
for these you should assume that `id` will be a number already. the type checking should occur before calling this, so that logic would be in the component